### DISCUSSION TITLE
[Bug report] Uncaught Exception - Saving Author Details

### DISCUSSION THREAD

**Discussion by example-user30** [GetPublii/Publii#1626] [Bug Report] [answered]
### Operating system

Edition	Windows 11 Enterprise
Version	21H2
Installed on	‎2/‎1/‎2023
OS build	22000.3147
Experience	Windows Feature Experience Pack 1000.22004.1000.0

### Publii version

0.46.0 - latest build

### Issue type

Application

### Bug description

Yesterday I was editing Author information without an issue.  Last night Windows updated and my PC rebooted - Publii was open at the time.  I relay this information not knowing if it contributed to the issue, as the issue is new this morning.
When I edit the Author information for my one and only author I get an uncaught exception on a missing javascript file.  I know this is related to the specific website I am working on - I created a dummy website and I can edit the Author information without issue - so I would imagine this would be almost impossible to recreate.  Sorry.
See the image attached for the error.

![error message](https://github.com/user-attachments/assets/fb6fe808-1a96-43d1-b8b8-ad4d31f6b058)

Interestingly - even though there is an error Publii still saves the changes.

### Steps to reproduce

I was able to reproduce the issue.  1) Create a new site.  2) go to Edit Author.  3) drag and drop an image onto the featured image.  4) Save.  I get the error here.

Note that I removed Publii completely from my PC using BCU, installed from the 0.46.0 download file and was still able to reproduce this error.

### Relevant log output

```shell
I could not find a log for this specific issue.
```

**Comment by dziudek** [comment]
@example-user30 thank you for finding this - it will be fixed in hot fix release tonight - it is due usage of wrong function name in the code

**Comment by bobmitro** [comment, marked as answer]
Build [16910](https://getpublii.com/download/) has just been released, which fixes this issue.

**Reply by TabiZz** [reply]
It also fixes the fact that we couldn't add avatar image! Thanks for you reactivity
